app/controllers/workshops_controller.rb

```diff
@@ -20,7 +20,8 @@ def show
     @workshop = Workshop.find_by(id: params[:id])
     @user = current_user.presence || User.new
     @registrant = Registrant.new
-    @registrants = find_registrants(@workshop.id)
+    @registrant_local_school = Registrant.where(workshop_id: params[:id]).where('registrants.local_school_id = ?', @user.local_school_id)
+    # @registrants = @workshop.registrants.where(local_school_id: current_user.local_school_id) if @user.local_school_admin?
```

Comment:
We need this?

Reply:
We are using it for the table I added

Reply:
@jhiatt It's commented out - so right now it's not being used anywhere